tasks/managed/collect-mrrc-params/collect-mrrc-params.yaml

```
@@ -50,13 +55,23 @@ spec:
         echo "export MRRC_PRODUCT_NAME=$productName" >> "$MRRC_ENV_FILE_PATH"
         echo "export MRRC_PRODUCT_VERSION=$productVersion" >> "$MRRC_ENV_FILE_PATH"
 
+        sign_key="$(jq -re '.mrrc.signing.signKey' "$DATA_FILE")"
+        echo "export MRRC_SIGN_KEY=$sign_key" >> "$MRRC_ENV_FILE_PATH"
+
         SNAPSHOT_PATH="$WORK_DIR/$(params.snapshotPath)"
         zipRegistries="$(jq -re '[.components[].containerImage] | join("%")' "$SNAPSHOT_PATH")"
         echo "export MRRC_ZIP_REGISTRY=$zipRegistries" >> "$MRRC_ENV_FILE_PATH"
 
         awsSecret="$(jq -re '.mrrc.awsSecret' $DATA_FILE)"
         echo -n "$awsSecret" > "$(results.charonAWSSecret.path)"
 
+        sign_ca_secret="$(jq -re '.mrrc.signing.signCASecret' "$DATA_FILE")"
+        echo -n "$sign_ca_secret" > "$(results.charonSignCASecret.path)"
+
+        AUTHOR=$(jq -r '.status.attribution.author' "$WORK_DIR/$(params.releasePath)")
+        if [[ ${AUTHOR} == "" ]] ; then exit 1 ; fi
```

If the field is not even defined (like in your test where you just create `{}`), the resulting AUTHOR string will be null and the script will carry on. Probably not the intention? So I'd recommend adding a default:
```suggestion
        AUTHOR=$(jq -r '.status.attribution.author // ""' "$WORK_DIR/$(params.releasePath)")
        if [[ ${AUTHOR} == "" ]] ; then exit 1 ; fi
```

---

Sure and fixed.